ui/contexts/metamask-notifications/metamask-notifications.tsx

```diff
@@ -44,8 +43,6 @@ export const MetamaskNotificationsProvider: React.FC = ({ children }) => {
     [isProfileSyncingEnabled, isNotificationsEnabled],
   );
 
-  useAccountSyncingEffect();
```

Comment:
Nice!

Note for notifications team - clean up this context. I really doubt we actually need a useContext and Provider.